source/features/create-release-shortcut.tsx

```diff
@@ -17,14 +17,11 @@ void features.add(import.meta.url, {
 	include: [
 		pageDetect.isReleasesOrTags,
 	],
-	deduplicate: 'has-rgh',
 	init,
 }, {
 	include: [
-		pageDetect.isReleasesOrTags, // If the release couldn't be published, GitHub changes the url to /releases while still being on the "New release" page
```

Comment:
This causes an error on every `isReleasesOrTags` page. I'd rather have a feature that uses `replaceState` to `location.href` instead